core/src/main/java/io/kestra/core/exceptions/MigrationRequiredException.java

```diff
@@ -0,0 +1,16 @@
+package io.kestra.core.exceptions;
+
+import lombok.Getter;
+
+import java.io.IOException;
+import java.io.Serial;
+
+@Getter
+public class MigrationRequiredException extends RuntimeException {
+    @Serial
+    private static final long serialVersionUID = 1L;
+
+    public MigrationRequiredException(String kind) {
+        super("It looks like the " + kind + " migration hasn't been run, please check out the changelog for instructions to do so.");
```

Comment:
I think it would be better to explain that the migration needs to be done via the command instead of referring to the changelog so a user didn't have to read the doc to know what to do.
Especialy because we have nothing that is named a migration yet.